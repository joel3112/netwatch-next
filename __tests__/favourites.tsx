import { render } from '@testing-library/react';
import FavouritesPage from '@/pages/favourites';

describe('Tests favourites page', () => {
  test('renders page correctly', () => {
    const { container } = render(<FavouritesPage />);

    expect(container).toMatchSnapshot();
  });
});
