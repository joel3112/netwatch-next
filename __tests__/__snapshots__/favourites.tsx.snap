// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests favourites page renders page correctly 1`] = `
<div>
  <div
    class="wrapper hasMargins"
  >
    <div
      class="wrapper"
      style="margin-top: 30px;"
    >
      <h2
        class="wrapper"
      >
        favourites.title
      </h2>
      <div
        class="wrapper"
      >
        <div
          class="MuiGrid-root MuiGrid-container wrapper grid css-lrun27-MuiGrid-root"
          role="grid"
        />
      </div>
      <style>
        
        .wrapper {
          display: flex;
          flex-direction: column;
          flex-wrap: nowrap;
          gap: 20px;
          
          
        }
      
      </style>
    </div>
  </div>
</div>
`;
